.travis.yml

```diff
@@ -9,6 +9,13 @@ rvm:
   - 2.3.0
   - ruby-head
   - rbx-2
+  - jruby-9.0.4.0
+  - jruby-9.0.5.0
```

Comment:
@groyoh 
`jruby-9xxx` and `jruby-91xx` (coming soon) are two different `ruby` versions. but for now, having just one `jruby-9xxx` should be enough, i agree.

regarding the `include` - you mean that we won't run `jruby` against Rails 4.0 ?